content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -299,6 +299,21 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+
+### container.apparmor.security.beta.kubernetes.io/* (beta) {#container-apparmor-security-beta-kubernetes-io}
+
+Type: Annotation
+
+Example: `container.apparmor.security.beta.kubernetes.io/my-container: my-custom-profile`
+
+Used on: Pods
+
+This annotation allows you to specify the AppArmor security profile for a container within a
+Kubernetes pod. AppArmor is a Linux kernel security module that restricts the capabilities
```

Comment:
```suggestion
Kubernetes pod. 
```
It would be better to not add extra details when user will find the information in the link.